src/AppInstallerCLICore/COMContext.h

```diff
@@ -56,6 +56,8 @@ namespace AppInstaller
         //Execution::Context
         void SetExecutionStage(CLI::Workflow::ExecutionStage executionPhase, bool);
 
+        CLI::Workflow::ExecutionStage GetExecutionStage() const { return m_executionStage; }

```

Comment:
This should be on the base `Context`; not sure why you moved it.

Reply:
ComContext apparently has its own m_executionStage that it updates in SetExecutionStage, base class doesn't get called, so getting it from base class is useless apparently.